cumulusci/tasks/github/merge.py

```diff
@@ -28,8 +28,11 @@ class MergeBranch(BaseGithubTask):
         "branch_prefix": {
             "description": "A list of prefixes of branches that should receive the merge.  Defaults to project__git__prefix_feature"
         },
+        "skip_future_releases": {
+            "description": "If true, then exclude branches that start with the branch prefix if they are not for the lowest release number. Defaults to True."
+        },
```

Comment:
> If true, then exclude branches that start with the branch prefix if they...

Which prefix is being referred to: feature, release?

Reply:
The one named in the task option called branch_prefix. It is typically set to the feature branch prefix.